### ISSUE TITLE
Squished screenshots and logos

### ISSUE THREAD

**Issue by duiz** [remeh/mehstation#3]
I've noticed that mehstation often resizes logos and screenshots vertically making everything have a rather sandwiched look. Any chance in the future we could get a maintain original aspect ratio on the scraped content to avoid this?

**Comment by remeh** [comment]
Hi,
Yes that's definitely something which can be done and which would not need many changes.

**Comment by duiz** [comment]
Will you maybe set up a forum on your website so we can make suggestions and give feedback to your development? mehstation is one of the coolest frontends I ever used and I would love see it grow :)

**Comment by remeh** [comment]
Hey thanks, means a lot :)
Yeah I definitely need to setup a forum and complete the homepage of mehstation before the official 1.0 release...

**Comment by HerbFargus** [comment]
I just barely came across your project. looks very promising., to add to this issue the same thing happens with the default platform text

![mehstation](https://cloud.githubusercontent.com/assets/10035308/13731274/a2d4f542-e92a-11e5-9ea1-3195efae55ea.png)

Is  there any possibility in the future for more dynamic theming so that the user can natively control the layouts and aspect ratios of the boxart (similar to emulationstation?) I presume the layout is currently hardcoded

**Comment by remeh** [comment]
Image ratio fixed:
https://github.com/remeh/mehstation/commit/cd61d0dcb4fb6687230f70246f5534ad3f1ffac4

@duiz for the release of the 1.1 (june or july 2016), I've digged in some forums solution for mehstation but they actually scare me (exploit, account leaks, ...). So I've opened a subreddit on https://reddit.com/r/mehstation hoping it'll do the job, at least for the beginning. I'll announce it with the next release.

@HerbFargus sorry I did forgot about the second part of your post: it's right that the layout is currently hardcoded and I don't to plan to code something themable _for the next release_, but, after that, it is something conceivable and not too complicated with the current code base, in some extent. That's something I've already considered and not put aside.

Thanks for the feedbacks :)
